code/pizzabot.js

```diff
@@ -1,7 +1,61 @@
 const vegetarian = "Vegetarian Pizza"
 const hawaiian = "Hawaiian Pizza"
 const pepperoni = "Pepperoni Pizza"
-
 const pizzaPrice = 80
 
-//Put your Javscript code here:
+console.log(vegetarian);
+console.log(hawaiian);
+console.log(pepperoni);
+console.log(pizzaPrice);
```

Comment:
We can delete the `console.log()`'s. We use them when developing the page, but we don't want them there once the project is done!